### PR TITLE
Benchmark automation

### PR DESCRIPTION
**Pull request by eggrobin**

### CODE REVIEW

File: benchmark_automation/benchmark_automation.cs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            CreateNoWindow = true
+          }
+        };
+        process.Start();
+        while (!process.StandardOutput.EndOfStream) {
+          String line = process.StandardOutput.ReadLine();
+          String[] words =
+              line.Split(separator : new Char[]{' '},
+                         options   : StringSplitOptions.RemoveEmptyEntries);
+          if (words[0].EndsWith("_mean")) {
```
Comment (pleroy) [inline benchmark_automation/benchmark_automation.cs]: This mean/stddev analysis is only going to work for benchmarks that have a repetition flag.  For the others you want to get the (single) result line too.

File: benchmark_automation/benchmark_automation.cs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            String benchmark_name = words[0].Substring(0, words[0].Length - 5);
```
Comment (pleroy) [inline benchmark_automation/benchmark_automation.cs]: If we gave a name to "_mean" we could use its length instead of 5.  Same for 7.

Comment (eggrobin) [inline benchmark_automation/benchmark_automation.cs]: Done.

File: benchmark_automation/benchmark_automation.cs (hunk 1 above)
Comment (eggrobin) [inline benchmark_automation/benchmark_automation.cs]: Done.

Comment (eggrobin) [inline benchmark_automation/benchmark_automation.cs]: Not that it matters with the commands we currently run.

**Pull request merged**